recipes/metatensor-torch/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "metatensor-torch" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # pretend to use the same source as the Python package, so we can keep the same
+  # version number up to date automatically with the help of the conda-forge bot
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/metatensor_torch-{{ version }}.tar.gz
+  sha256: d5180d90e9645d604d7e4817737298565d9d4a95aff7ad9e6407d3b9a5376b13
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - libmetatensor-torch
+    - python-metatensor-torch
+    - python-metatensor-operations
+    - python-metatensor-learn
+
+test:
+  commands:
+    # check if libmetatensor_torch files exists
+    - test -f $PREFIX/lib/libmetatensor_torch$SHLIB_EXT  # [unix]
```

Comment:
Looks like this is a repackage, right? We should build it from source for conda-forge. Repackages are only accepted in extremely exceptional cases. Even those we are slowly removing and building from source.

Reply:
We never defined a policy on metapackages, or if we did, I don't recall it. Pinging @conda-forge/core here for someone else's input.

My personal opinion is to always use environment instead of meta-packages. Meta-packages only adds another layer to an already complex system. Having a proper env file and using it with a lockfile locally, depending on the application usually works wonders.

Reply:
It looks like @isuruf is the best person to review this one then ;-p